Dockerfile

```diff
@@ -19,17 +19,27 @@ ENV PATH="$PATH:/usr/share/julia-1.5.3/bin" \
     FLASK_APP=pyreisejl/utility/app.py
 
 WORKDIR /app
-COPY . .
+COPY requirements.txt .
+RUN pip install -U pip; pip install -r requirements.txt
+
+COPY Project.toml .
+COPY Manifest.toml .
+RUN mkdir src && touch src/REISE.jl
 
 RUN julia -e 'using Pkg; \
 	Pkg.activate("."); \
 	Pkg.instantiate(); \
 	Pkg.add("Gurobi"); \
+	Pkg.add("GLPK")'
+
+COPY src src
+
+RUN julia -e 'using Pkg; \
+	Pkg.activate("."); \
```

Comment:
Do we need to invoke `Pkg.activate(".")` twice?

Reply:
Nope, thanks.